dash/frontend/src/app/modules/private/pages/reports/vulnerability-difference-by-date/vulnerability-difference-by-date.component.ts

```diff
@@ -116,8 +127,8 @@ export class VulnerabilityDifferenceByDateComponent implements OnInit, OnDestroy
       namespaces: [[]],
       severityLevels: [[]],
       fixAvailable: [],
-      startDate: ['', [this.customValidatorService.dateInPastOrToday]],
-      endDate: ['', [this.customValidatorService.dateInPastOrToday]]
+      startDate: [this.twoDaysAgo, [Validators.required, this.customValidatorService.dateInPastOrToday]],
+      endDate: [this.yesterday, [Validators.required, this.customValidatorService.dateInPastOrToday]]
```

Comment:
We should probably have backend enforcement/validation or defaulting of these dates as well, as it looks like the query will bomb without them.

Reply:
Yeah it will 😅